package.json

```diff
@@ -60,9 +60,12 @@
     "publish": "check-installed-dependencies && npm run compile && npm test && npm run props-table && lerna publish",
     "start": "cd packages/terra-consumer-site && npm run start",
     "start:express": "node scripts/express/app.js",
+    "storybook": "start-storybook -p 8081 -c .storybook",
     "test": "npm run jest && npm run nightwatch"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^3.2.14",
```

Comment:
You may want to update to `3.3.0-alpha.4`. Version `3.2` had issues with loading JSONP which is needed for requiring translations. This issue was fixed in v3.3.0 alphas. More info: https://github.com/storybooks/storybook/issues/1411